charts/datadog/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Datadog changelog
 
+## 2.33.5
+
+* Make the DCA leader election ConfigMap name depend on the Helm release name.
```

Comment:
Let's mention the minimum required version of the cluster agent

Reply:
It should work with previous version. but it will not take the information

Reply:
```suggestion
* Make the DCA leader election ConfigMap name depend on the Helm release name. (Requires DCA 1.21+)
```